### PR TITLE
Code style updates

### PR DESCRIPTION
**Pull request by helena-intel**
- black formatting
- minor refactoring to use utils function where useful
- remove check for installation of OpenVINO 2021.4 from some notebooks (is assumed now)
- replaced `/path/to/model_optimizer` with `mo`
- added parameters to many function calls
- modified some descriptions and headers for clarification and consistency
- added semicolons after plt.imshow() calls to prevent printing matplotlib axis info

Not intended to be complete - incremental improvements

More readable diff attached as PDF (created with nbdime and ansi2html)
[diff.pdf](https://github.com/openvinotoolkit/openvino_notebooks/files/7147820/diff.pdf)

### CODE REVIEW

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 1]
```diff
@@ -2,13 +2,16 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "id": "b875b31c",
```
Comment (ryanloney) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:5]: @helena-intel  just curious, but why do these ids show up now? Do we use them for something?

**Review by ryanloney — APPROVED** [review]
I tried each notebook and reviewed the changes. LGTM

File: notebooks/002-openvino-api/002-openvino-api.ipynb (hunk 1 above)
Comment (helena-intel) [reply notebooks/002-openvino-api/002-openvino-api.ipynb:5]: This is a change in the notebook format. This is not the best link but gives some info and issues https://github.com/jupyterlab/jupyterlab/issues/9729#issuecomment-803197368

**Comment by helena-intel** [comment]
> I tried each notebook and reviewed the changes. LGTM

Thanks for trying them all!

**Pull request merged**